resources/assets/lib/beatmap-discussions/new-reply.tsx

```diff
@@ -253,9 +259,10 @@ export class NewReply extends React.Component<Props> {
   }
 
   @action
-  private readonly setMessage = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    this.message = e.target.value;
-  };
+  private setMessage(message: string) {
+    this.message = message;
+    this.storeMessage();
+  }
```

Comment:
I've since discovered if a getter is decorated as `computed`, the corresponding setter automatically runs as an action without needing the decoration 👀

Reply:
but there's no getter here 🤔

Reply:
but there could be <_<

also, `storeMessage` should be rolled up into `setMessage`